implementations/rust/ockam/ockam_identity/src/trust_context.rs

```diff
@@ -0,0 +1,119 @@
+use async_trait::async_trait;
+
+use crate::{credential::Credential, error::IdentityError, Identity, PublicIdentity};
+use ockam_core::compat::{boxed::Box, string::String, sync::Arc};
+
+/// Trust Context is a set of information about a trusted authority
```

Comment:
A trust context defines which authorities are trusted to attest to which attributes, within a context.

Our first implementation assumes that there is only one authority and it is trusted to attest to all attributes within this context.